src/dali/plugin/input/rest/coinbase_pro.py

```diff
@@ -334,8 +334,8 @@ def _process_fills(self, transaction: Any, in_transaction_list: List[InTransacti
                     transaction_type="Sell",
                     spot_price=str(from_currency_price),
                     crypto_out_no_fee=str(from_currency_size),
-                    crypto_fee="0",
-                    crypto_out_with_fee=None,
+                    crypto_fee=str(crypto_fee),
+                    crypto_out_with_fee=str(from_currency_size+crypto_fee),
```

Comment:
You can leave `crypto_out_with_fee=None`, because RP2 computes it automatically as` crypto_out_no_fee + crypto fee` (see https://github.com/eprbell/rp2/blob/main/src/rp2/out_transaction.py#L69).